atari-game-play/steps/build_dqn.py

```diff
@@ -0,0 +1,27 @@
+from .configs import PreTrainingConfigs
+from dqn.model import *

```

Comment:
as Hamza mentioned elsewhere, wildcard imports are usually not considered good practice as they can lead to unexpected behavior.